utils/web_scraper.py

```
@@ -18,9 +18,54 @@
+import importlib.util
 import json
+import sys
 
+# ==================================================
+# 增加缺失的 bilibili_api 数据文件修复逻辑，防止打包后环境缺失导致的错误
+# ==================================================
+def fix_missing_bilibili_api_json():
+    """
+    修复缺失的 bilibili_api 数据文件
+
+    当打包后的环境中缺少 bilibili_api 数据文件时，尝试修复该问题。
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)